docs/sphinx/source/whatsnew/v0.8.0.rst

```diff
@@ -35,6 +39,8 @@ Testing
   applied to functions that require args or kwargs. (:pull:`973`)
 * Test added for :py:class:`pvlib.modelchain.ModelChain` to confirm ValueError when
   ``ac_model`` is an invalid string. (:pull:`886`)
+* Tests added for :py:func:`pvlib.pvsystem.irradiance_losses_pvsyst` and
+  :py:meth:`pvlib.pvsystem.PVSystem.irradiance_losses_pvsyst`
```

Comment:
I'm not opposed to having these lines in whatsnew, just noting that we haven't explicitly called out adding tests for new functions - flagging it in case other maintainers have an opinion here.

Reply:
I agree with @wholmgren we can omit these lines from whatsnew